scripts/release/build_images.py

```diff
@@ -0,0 +1,159 @@
+# This file is the new Sonar
+import base64
+from typing import Dict
+
+import boto3
+import python_on_whales
```

Comment:
do we describe/compare python_on_whales in the td compared to other wrappers?

Reply:
I'll add a section, but TL;DR it is lower level and have 100% parity with docker cli. Especially for using buildx